### DISCUSSION TITLE
Async client for python lib

### DISCUSSION THREAD

**Discussion by KozyarValeriy** [flipt-io/flipt#2792] [General]
Hi there, 
We are currently using version 0.2.17, which has an asynchronous client. Due to the use of the older version, we would like to upgrade to the latest version, but it doesn't have an asynchronous client. 

Are you considering adding an asynchronous client to Python lib? Or can I submit a pull request with the addition of the client on my own?

**Comment by GeorgeMac** [comment]
Hey @KozyarValeriy 👋 

We 💯% welcome contributions to add an asynchronous Python lib, if you have the time to contribute 🙏 
That would be amazing.

Apologies, this got lost in the migration away from our previously auto-generated clients.

The root of this new v1 client is here https://github.com/flipt-io/flipt-server-sdks/tree/main/flipt-python/flipt
As a practice, we're only focussing on the evaluation section of our API for these new clients, at least in the meantime.

**Comment by KozyarValeriy** [comment]
Good, I will try to make a pull request. 

Do you mind if I replace unittest with pytest and add ruff linter and mypy?
Additionally, can I move the code from the __init__ file into separate files (I will add the import statement in the __init__ in order to maintain backward compatibility)?

**Reply by GeorgeMac** [reply]
We welcome all suggestions! That sounds great.

**Reply by GeorgeMac** [reply]
Im so slow the guys got to it already. Thanks for this!
